### PR TITLE
Feature/lp

### LINKED ISSUE #70: The header section is missing

### LINKED ISSUE #71: The Energy Cells Explained section is missing

### LINKED ISSUE #72: The tool + features section is missing

### LINKED ISSUE #73: The projects section is missing

### LINKED ISSUE #74: The about us section is missing

### LINKED ISSUE #75: The footer is missing

### LINKED ISSUE #78: Favicon is missing

### LINKED ISSUE #84: Imprint and Privacy pages are missing
@catcad @Bachibouzouk  should we use the text from the RLI website? If yes, there are still a few things we should change though: there won't be any tracking tool, right? Will there be some cookies on the website?

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Fix #84, fix #78, fix #75, fix #74, fix #73 fix #72, fix #71, fix #70 

**Changes proposed in this pull request**:
- Your_changes

The following steps were realized, as well (if applies):
- [x] Apply black (`black . --exclude docs/`)
- [x] Update the CHANGELOG.md

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by Bachibouzouk** [comment]
@bmlancien you don't need to merge branch dev into this branch as no one else as you is working on these files, therefore there are no potential conflicts :)

**Comment by bmlancien** [comment]
> you don't need to merge branch dev into this branch as no one else as you is working on these files, therefore there are no potential conflicts :)

Alright, good to know. I did it just in case :)

File: website/static/favicon/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Bachibouzouk) [inline website/static/favicon/browserconfig.xml:1]: what is the use of this file @bmlancien ?

**Comment by Bachibouzouk** [comment]
Thanks @bmlancien - I merged it, you can open a new branch from dev (need to run `git pull` locally on your branch `dev` before creating a new branch out of it) if you have more things to add: `git checkout -b feature/<name of your branch>`

File: website/static/favicon/browserconfig.xml (hunk 1 above)
Comment (bmlancien) [reply website/static/favicon/browserconfig.xml:1]: I think this is for displaying a favicon for IE on WIndows 8 and 10

**Pull request merged**